### DISCUSSION TITLE
how to bind Neovim LSP completion and path completion in normal mode to Up And Down Arrows

### DISCUSSION THREAD

**Discussion by xander412** [NvChad/NvChad#2943] [Q&A] [answered]
I have been using VIM for a very very long time in a distributed remote environment , I am used to Up And Down Arrows for Auto Completion. Please let me know how to bind these keys. Thank You

**Comment by siduck** [comment]
just  override cmp config, look at the last example https://nvchad.com/docs/config/plugins and apply it to cmp

**Reply by xander412** [reply]
@siduck . Thank you very much for the help. I really appreciate it.

**Comment by xander412** [comment, marked as answer]
Change mapping in lua/plugins/configs/cmp.lua:94 . Referred commit 85b9f5686991a8b98c4980747f484ce155c959d0.

**Reply by siduck** [reply]
hmm why's that commit related here?

**Reply by xander412** [reply]
I specified the commit because, in future someone who sees this answer might see different code in the same file and line (changes might happen). So, I specified the commit.
